utils/wasm-instrument/Cargo.toml

```diff
@@ -17,6 +17,10 @@ gwasm-instrument = { workspace = true, features = ["sign_ext"] }
 derive_more.workspace = true
 enum-iterator.workspace = true
 
+# TODO: remove when tooling is updated
```

Comment:
Before updating tooling, it makes sense to replace `parity-wasm` with `gear-wasm` in all our crates (except wasmi 0.13.2 and other substrate-related dependencies).

Also try applying existing patches to our tooling as git dependency:
- https://github.com/gear-tech/parity-wasm/pull/4
- https://github.com/gear-tech/wasm-instrument/pull/8

Reply:
It should also be under `[target.'cfg(force_wasmer_cranelift_i_know_what_i_do)'.dependencies]`.